src/eradiate/radprops/_afgl1986.py

```diff
@@ -4,20 +4,23 @@
 
 from __future__ import annotations
 
-import functools
 import typing as t
+import warnings
+from functools import partial, singledispatchmethod
+from os import PathLike
```

Comment:
If it's for typing, please use our `PathLike` construct.

Reply:
Outdated